### ISSUE TITLE
When an object inherits from a class the diagram is not displaying this relationship

### ISSUE THREAD

**Issue by tesonep** [uqbar-project/wollok#457]
```
/*
 * This is an example wollok hello world
 */
object ex {
	method sayHi(){
		console.println('Hello Wollok!')
	}
}

object asd inherits A{
	
}

class A {
	method something(){
		return this.doMagic()
	}

	method doMagic(){
		return 23
	}
}

class B inherits A {
	override method doMagic(){
		return 17
	}
}
````

It's working as a charm, but the diagrama is not drawing correctly. Shows the relationship between A and B, but not between *asd* and A

**Comment by javierfernandes** [comment]
Yeah, this was reported here 
#266

There's also another issue to show other types of relations
#345

**Comment by javierfernandes** [comment]
Done 
<img width="271" alt="screen shot 2015-12-07 at 17 50 43" src="https://cloud.githubusercontent.com/assets/4428120/11640051/41370b84-9d0f-11e5-8019-e78cfeeea7a8.png">

**Comment by tesonep** [comment]
Grosso

On Mon, Dec 7, 2015 at 6:21 PM, javierfernandes <notifications@github.com>
wrote:

> Closed #457 <https://github.com/uqbar-project/wollok/issues/457>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uqbar-project/wollok/issues/457#event-484911611>.
>



-- 
Pablo Tesone.
tesonep@gmail.com
